modules/schematics-core/utility/ngrx-utils.ts

```diff
@@ -152,11 +154,13 @@ export function addReducerToActionReducerMap(
 
   let node = actionReducerMap.initializer;
 
+  const state =
+    options.plural === undefined
```

Comment:
This check isn't sufficient, if we set `plural` to `false` we would still pluralize the name and this might be confusing 😄

Reply:
Yes i was thinking about same, after i went to sleep. thanks for catching it.